feathr_project/feathr/lookup_feature.py

```diff
@@ -0,0 +1,56 @@
+from copy import copy, deepcopy
+from typing import List, Optional, Union
+
+from jinja2 import Template
+
+from feathr.dtype import FeatureType
+from feathr.feature import FeatureBase
+from feathr.transformation import RowTransformation
+from feathr.typed_key import DUMMY_KEY, TypedKey
+from feathr.aggregation import Aggregation
+
+class LookupFeature(FeatureBase):
+    """A lookup feature is a feature defined on top of two other features, i.e. using the feature value 
+    of the base feature as key, to lookup the feature value from the expansion feature.
```

Comment:
this is good description but due to the complexity of it, it's better to provide an example here.